demisto_sdk/commands/content_graph/parsers/integration_script.py

```diff
@@ -91,4 +91,4 @@ def subtype(self):
     def auto_update_docker_image(self):
         return (
             get_value(self.yml_data, "autoUpdateDockerImage", "")
-        ).lower() != "false"
+        ).lower() not in ("false", False)
```

Comment:
lets add a test that a yml provided with no input at all defaults to `auto_update_docker_image = True`